cmd/enterprise-portal/service/iam_model.fga.yml

```
@@ -0,0 +1,74 @@
+# To run this test suite:
+#
+#   go run github.com/openfga/cli/cmd/fga@latest model test --tests='cmd/enterprise-portal/service/iam_model.fga.yml'
+#
+# See https://openfga.dev/docs/modeling/testing
+
+model_file: ./iam_model.fga
+
+tuples:
+- user: user:018d21f2-04a6-7aaf-9f6f-6fc58c4187b9
+  relation: member
+  object: customer_admin:80ca12e2-54b4-448c-a61a-390b1a9c1224
+
+- user: user:617E04BC-905D-4543-94E7-55DCF7F5E92F
+  relation: member
+  object: customer_admin:4CDA7E9F-69D8-4594-9637-F964693ADFE5
+
```

don't we need to make it so members of a specific customer admin have access to specific subscriptions?
```suggestion
- user: customer_admin:80ca12e2-54b4-448c-a61a-390b1a9c1224#member
  relation: view
  object: subscription_cody_analytics:80ca12e2-54b4-448c-a61a-390b1a9c1224

- user: customer_admin:4CDA7E9F-69D8-4594-9637-F964693ADFE5#member
  relation: view
  object: subscription_cody_analytics:4CDA7E9F-69D8-4594-9637-F964693ADFE5

```

(is it expected that customer admin objects and subscription_cody_analytics objects have the same IDs?)